### ISSUE TITLE
Sample Code needs update 

### ISSUE THREAD

**Issue by dexterac** [PacktPublishing/Hands-On-Blockchain-for-Python-Developers#2]
I got the book in December and while running the code as listed in the book these are the errors

```
    Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> project:/contracts/donation.sol

,Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
 --> project:/contracts/donation.sol:9:3:
  |
9 |   constructor() public {
  |   ^ (Relevant source part starts here and spans across multiple lines).


TypeError: Type address is not implicitly convertible to expected type address payable.
  --> project:/contracts/donation.sol:10:15:
   |
10 |     donatee = msg.sender;
   |               ^^^^^^^^^^
```

**Comment by arjunaskykok** [comment]
Hi @dexterac ,

I'm the author of the book. I'm regretful to inform you that the book has shown its age. The Vyper has undergone many improvements. So the current Vyper version is vastly different than the Vyper version I used when I wrote the book.

My suggestion is to get the general idea from the book and adapt the code to the Vyper documentation:

https://vyper.readthedocs.io/en/latest/toctree.html
